src/util/kaldi-table-inl.h

```diff
@@ -178,25 +203,41 @@ template<class Holder>  class SequentialTableReaderScriptImpl:
       else  // orig_state_ was kLoadFailed, which only could have happened
         // if the user called FreeCurrent().
         KALDI_ERR << "You called Value() after FreeCurrent().";
-    } else if (state_ != kLoadSucceeded) {
+    } else if (state_ == kRangeExtractionFailed) {
+      KALDI_ERR << "Failed to load object from "
+                << PrintableRxfilename(data_rxfilename_) 
+                << " within range " << range_
```

Comment:
actually, it might be clearer to just print
data_rxfilename_[range]
i.e. to print square brackets.